routes/reading.js

```diff
@@ -55,10 +75,56 @@ router.get("/", function (req, res) {
   });
 });
 
+router.get("/feedback/:id",function(req,res)
+{
+  res.render("test",{id:req.params.id});
+});
+
 router.get("/load/:id", function (req, res) {
   Node.findOne({ _id: req.params.id }).then((node) => {
     res.send(node);
   });
 });
 
+  router.post("/new/:id",  [
+
+  body('rating').isNumeric({ min: 0,max :5})
+], function(req,res)
+  {
+    const errors=validationResult(req);
+    console.log(errors);
+    if (!errors.isEmpty()) {
+
+        res.send("Invalid Rating");
+      console.log(errors);
+      //  console.log(req.body.nodes[0].content);
+
+    }
+else
+{
+     var num1=Number(req.body.rating);
+
+let id=req.param.id;
+
+Blog.findByIdAndUpdate(
+    req.params.id,
+   {$inc: {rating:num1 }},
+
+   {upsert:false},
+
+   function(err, document){
+
```

Comment:
You can write a simple console.log here cause otherwise there's no purpose of an error handling function.